.github/workflows/label.yml

```diff
@@ -0,0 +1,71 @@
+name: Auto Label New Issues, PRs, and Discussions
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+  discussion:
+    types: [created]
+
+jobs:
+  add-label:
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+      discussions: write
+      pull-requests: write
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      REPO_OWNER: ${{ github.repository_owner }}
+      REPO_NAME: ${{ github.event.repository.name }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Add context-v2 label to new issue
+        id: label-issue
```

Comment:
Let's remove the `id`s of steps that we don't need to keep things short. I think only the `label-data` step ID is necessary, right?

Reply:
Done in 431d16811acfae01b8078f52c147388855187648.